src/main/webapp/resources/js/viewer/controller.js

```diff
@@ -204,15 +205,18 @@ function Controller(bookID, accessible_modes, canvasID, regionColors, colors, gl
 				} else {
 					this._requestEmptySegmentation();
 				}
-				pages.forEach(page => { _gui.addPageStatus(page, PageStatus.SERVERSAVED)});
+
```

Comment:
Why was this removed? After removing the icon for pages with a saved annotation isn't shown when the page hasn't been selected yet. Am I missing something?

Reply:
No, you're absolutely right. Added some functions to the loop which I later removed again and accidentally removed outer loop with it.